docs/high-performance-order-storage/guide-large-store.md

```diff
@@ -0,0 +1,134 @@
+# A large store's guide to enable HPOS on WooCommerce
+
+Do you run a high-volume WooCommerce store? Are you about to embark on the task to enable HPOS on said store? As we move more and more high-volume store to HPOS, we decided to document the guidelines for those of you who may be faced with this task.
```

Comment:
For SEO reasons and also for legibility, let's disambiguate the HPOS acronym the first time we use it on a page:

> Do you run a high-volume WooCommerce store? Are you about to embark on the task to enable High Performance Order Storage (HPOS) on said store?